### PR TITLE
Home work05

### PR DESCRIPTION
**Pull request by DmitryHV**
Check Homework5, please.

### CODE REVIEW

File: FreeITCourse/src/lesson05/MetodTask21.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+// Задание 21
+// Напишите три цикла выполняющих многократное сложение строк,
+// один с помощью оператора сложения и String,
+// другой с помощью StringBuilder и метода append,
+// а также аналогино для StringBuffer.
+// Сравните скорость их выполнения.
+public class MetodTask21 {
+    public static void main(String[] args) {
```
Comment (TheFallen058) [inline FreeITCourse/src/lesson05/MetodTask21.java:15]: please use
System.nanoTime()
for getting more correct time intervals, it should make comparison easier.
run application at least 3 times and give some description what is more efficient

File: FreeITCourse/src/lesson05/MetodTask21.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        for (int i = 0; i < 30000 ; i++) {
+            str += addStr;
+        }
+        Date date2 = new Date();
+        long result1 = date2.getTime() - date1.getTime();
+        System.out.println("Total time for String + equals: " + result1 + " milisecond(s)");
+
+// check StringBuilder
+
+        StringBuilder strB = new StringBuilder("Check  of add");
```
Comment (TheFallen058) [inline FreeITCourse/src/lesson05/MetodTask21.java]: no need to cut variable name in this case. stringBuilder will be better

File: FreeITCourse/src/lesson05/MetodTask21.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        for (int i = 0; i < 30000 ; i++) {
+            strB.append(addStr);
+        }
+        Date date4 = new Date();
+        long result2 = date4.getTime() - date3.getTime();
+        System.out.println("Total time for StringBuilder equals: " + result2 + " milisecond(s)");
+
+// check StringBuffer
+
+        StringBuffer sb = new StringBuffer("Check  of add");
```
Comment (TheFallen058) [inline FreeITCourse/src/lesson05/MetodTask21.java]: no need to cut variable name in this case. stringBuffer will be better

**Review by TheFallen058 — COMMENTED** [review]
Please check my comments
Generally good work

File: FreeITCourse/src/lesson05/MetodTask21.java (hunk 2 above)
Comment (DmitryHV) [reply FreeITCourse/src/lesson05/MetodTask21.java]: corrected

File: FreeITCourse/src/lesson05/MetodTask21.java (hunk 3 above)
Comment (DmitryHV) [reply FreeITCourse/src/lesson05/MetodTask21.java]: corrected

File: FreeITCourse/src/lesson05/MetodTask21.java (hunk 1 above)
Comment (DmitryHV) [reply FreeITCourse/src/lesson05/MetodTask21.java:15]: corrected

**Pull request merged**